app/src/ApiCalendar.js

```diff
@@ -0,0 +1,319 @@
+import dotenv from "dotenv";
+require("dotenv").config();
+const result = dotenv.config();
+
+// if (result.error) {
+//   throw result.error;
+// }
+
+// console.log(result.parsed);
+// console.log("sillytestvar", process.env.SILLYTESTVAR);
+
+// import env from "react-dotenv";
+
+//Credit: https://github.com/Kubessandra/react-google-calendar-api
+const CLIENT_ID =
+  "60497895153-iiq79i18jc265rme5l0ojvagac8c622e.apps.googleusercontent.com";
+const API_KEY = "AIzaSyC5l6zqOjiHW5f1p5lp3A196lcWHtbokI8";
```

Comment:
I think those should go into your .env file

Reply:
You can load it from github secrets with `process.env.CLIENT_ID`, etc